src/NuGet.Services.AzureSearch/VersionList/Guard.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.AzureSearch
+{
+    internal static class Guard
+    {
+        /// <summary>
+        /// We could use <see cref="System.Diagnostics.Debug.Assert(bool, string)"/> here, but it's preferable
```

Comment:
This comment is worthy of &lt;remarks> section, not &lt;summary>. Maybe add couple of sentences about the purpose of the method?

Reply:
Done.